docker-compose.yml

```
@@ -1,9 +1,7 @@
 version: "3.8"
 services:
   inhouse_bot:
-    build:
-      context: .
-      dockerfile: ./Dockerfile
+    image: ghcr.io/adamschachne/inhouse_bot:latest
```

```suggestion
    image: inhouse_bot
```

I'd like simply `inhouse_bot` since it removes PII and `docker compose build` will name the image to inhouse_bot. It's cleaner for my local development.